jetty-ee9/jetty-ee9-openid/src/main/java/org/eclipse/jetty/ee9/security/openid/OpenIdAuthenticator.java

```diff
@@ -293,7 +294,7 @@ private void attemptLogoutRedirect(ServletRequest request)
             if (_logoutRedirectPath != null)
             {
                 StringBuilder sb = new StringBuilder(128);
-                URIUtil.appendSchemeHostPort(sb, request.getScheme(), request.getServerName(), request.getServerPort());
+                HttpScheme.appendNormalizedUri(sb, request.getScheme(), request.getServerName(), request.getServerPort());
```

Comment:
Use the method with more arguments.

Reply:
Not appropriate to use longer version here, as the next two lines build up a path.